front/app/containers/IdeasNewSurveyPage/IdeasNewSurveyForm/index.tsx

```diff
@@ -98,38 +95,6 @@ const IdeasNewSurveyForm = ({ project }: Props) => {
     : getCurrentPhase(phases?.data);
   const allowAnonymousPosting = phase?.attributes.allow_anonymous_participation;
 
-  const getApiErrorMessage: ApiErrorGetter = useCallback(
-    (error) => {
-      return (
-        messages[`api_error_${uiSchema?.options?.inputTerm}_${error}`] ||
-        messages[`api_error_${error}`] ||
-        messages[`api_error_invalid`]
-      );
-    },
-    [uiSchema]
-  );
-
-  const getAjvErrorMessage: AjvErrorGetter = useCallback(
-    (error) => {
-      return (
-        messages[
-          `ajv_error_${uiSchema?.options?.inputTerm}_${
-            getFieldNameFromPath(error.instancePath) ||
-            error?.params?.missingProperty
-          }_${error.keyword}`
-        ] ||
-        messages[
-          `ajv_error_${
-            getFieldNameFromPath(error.instancePath) ||
-            error?.params?.missingProperty
-          }_${error.keyword}`
-        ] ||
-        undefined
-      );
-    },
-    [uiSchema]
-  );
-
```

Comment:
There were no relevant api/ajv error messages for survey in the old IdeasNewPage folder defined. So this was already serving no use for surveys before.

I'll later check other places. I've noticed there's at least one other place where this is not used properly.